roles/check_system_static/tasks/main.yml

```diff
@@ -64,3 +64,42 @@
   fail:
     msg: "It is not recommended to set vm.overcommit_memory to 2, set it to 0 or 1."
   when: vm_overcommit_memory.stdout | int == 2
+
+- name: Check ansible version
+  fail:
+    msg: "Stop if ansible version is too low, make sure that the Ansible version is 2.4.2 or later, otherwise a compatibility issue occurs. Current ansible version is {{ ansible_version.full }}"
+  when: ansible_version.full | version_compare('2.4.2', '<')
+
+- name: Get if jmespath info[1]
+  shell: pip list | grep -i jmespath | wc -l
+  register: jmespath_exist
+
+- name: Get if jinja2 exist[1]
+  shell: pip list | grep -i jinja2 | wc -l
```

Comment:
Same as above.

But more, when  there is a package named: jinja2-abc the shell command will succeed.